cmd/commands/datasetIngestor.go

```diff
@@ -236,8 +238,7 @@ For Windows you need instead to specify -user username:password on the command l
 		// TODO ask archive system if sourcefolder is known to them. If yes no copy needed, otherwise
 		// a destination location is defined by the archive system
 		// for now let the user decide if he needs a copy
-
-		if nocopyFlag {
+		if nocopyFlag || beamlineAccount {
```

Comment:
why this?

Reply:
![Screenshot from 2024-08-26 14-34-26-mod](https://github.com/user-attachments/assets/c024fd4e-94c7-4cd9-9072-9ef755cbc0d6)
This is what confused me. Thinking about it now, it looks like it wants to set copy to false IF neither the copy nor nocopy are set, but at that point copy is by default false. The "checkCentralAvailability" already disables *checking* if the user is a beamline account, and since copyFlag is false by default... I think the previous version is correct. I'll restore it

Reply:
This code is from before the refactoring stuff (v3.0.0-alpha2)